README.md

```diff
@@ -307,6 +307,32 @@ By default, Periphery does not assume that declarations accessible by the Object
 
 Alternatively, the `--retain-objc-annotated` can be used to only retain declarations that are explicitly annotated with `@objc` or `@objcMembers`. Types that inherit `NSObject` are not retained unless they have explicit annotations. This option may uncover more unused code, but with the caveat that some of the results may be incorrect if the declaration is used in Objective-C code. To resolve these incorrect results, you must add an `@objc` annotation to the declaration.
 
+### SPI (System Programming Interface)
+
+Swift's `@_spi` attribute marks declarations as pseudo-private, making them accessible only to clients that explicitly import the SPI. While these declarations are technically `public`, they're intended for internal or restricted use.
+
+When using `--retain-public` for framework projects, all public declarations are retained, including those marked with `@_spi`. However, you may want to check for unused code within specific SPIs. The `--check-spi` option allows you to specify which SPIs should be checked for unused code even when `--retain-public` is enabled.
+
+For example, with `--retain-public --check-spi Internal`, Periphery will:
+- Retain regular `public` declarations
+- Retain `@_spi(Testing) public` declarations (different SPI)
+- **Check** `@_spi(Internal) public` declarations for unused code
+
+This is particularly useful for internal SPIs that should be audited for unused code while still retaining the framework's public API.
+
+**Configuration:**
```

Comment:
These `Configuration` and `Command line` sections feel a little redundant. I don't think we need to explain how to enable these options.